corehq/apps/userreports/static/userreports/js/configurable_reports_home.js

```diff
@@ -15,7 +15,7 @@ hqDefine("userreports/js/configurable_reports_home", [
     });
 
     $select.select2({
-        placeholder: gettext("Edit a report or data source"),
+        placeholder: gettext("Edit a custom web report or custom web report source"),
```

Comment:
I think this text change adds confusion rather than remove it since the reports and data sources edited on this page can be used in web or mobile.